compiler/fir/java/src/org/jetbrains/kotlin/fir/java/scopes/JavaOverrideChecker.kt

```diff
@@ -26,6 +26,19 @@ class JavaOverrideChecker internal constructor(
 ) : FirAbstractOverrideChecker() {
     private val context: ConeTypeContext = session.typeContext
 
+    private fun isEqualTypeProjections(
```

Comment:
I'd better call it `isEqualArrayElementTypeProjections`

Reply:
Sounds clearer, thanks!